firebase-app-distribution/firebase-app-distribution.gradle

```diff
@@ -48,8 +48,8 @@ dependencies {
     testImplementation 'junit:junit:4.13.2'
     testImplementation "org.robolectric:robolectric:$robolectricVersion"
     testImplementation "com.google.truth:truth:$googleTruthVersion"
-    testImplementation 'org.mockito:mockito-core:2.25.0'
-    androidTestImplementation "org.mockito:mockito-android:2.25.0"
+    testImplementation 'org.mockito:mockito-inline:3.4.0'
```

Comment:
curious, why did this import change?

Reply:
Was changed because mocking and verifying static methods was not available in the versions that were currently there, changed it to be able to use it for caclulateApkInternalCodeHash